mobile/src/main/res/xml/preferences.xml

```diff
@@ -72,6 +72,10 @@
             android:summary="%s"
             android:entries="@array/iconTypeValues"
             android:entryValues="@array/iconTypeValues" />
+        <Preference
+            android:clickable="true"
+            android:key="default_openhab_cleacache"
+            android:title="@string/mainmenu_openhab_clearcache" />
```

Comment:
Maybe add a summary like "Click to clear image cache" so the user knows that it will be clear immediately. Thats not a critical action, so maybe we dont need this hint.

Reply:
Hmm, I think that the title is already pretty clear in what this method does :)